testing-guidance/workflow-tests/android/android-lkg-workflows.md

```diff
@@ -0,0 +1,77 @@
+    "version_number": 1,
+    "version_name": "1.0.0",
+    "orientation": "landscape"
+},
+````
+* Build the default project on PC
+* Open editor and create a default level (`DefaultLevel`)
+* Create `DefaultProject\autoexec.cfg` file and include the following line to indicate the launcher to load the default level:
+````
+LoadLevel Levels/DefaultLevel/DefaultLevel.spawnable
+````
+* Run Asset Processor and wait until all assets have been processed.
+
+**Editor Platforms:**
+* Windows
+
+**Game Launcher Supported Platforms:**
+* Android
+
+**Product:** Proven that O3DE can be built on Android in different packaging modes and generate the right APK content and assets on each one.
+
+**Suggested Time Box:** 1 hour for requirements + 30 minutes per packaging mode.
+
+| Workflow                      | Requests                                                                                                                                                                                                                       | Things to Watch For                                                                                                                                                                                                                                                 |
+|-------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **Assets in APK**   | <ol><li>Uninstall `DefaultProject` app from device (if previously installed).</li><li>Follow the steps from https://www.o3de.org/docs/user-guide/platforms/android/generating_android_project_windows/ to build and deploy with the following alterations:<ol type="a"><li>Use `DefaultProject` you've created instead of `o3de-atom-sampleviewer`</li><li>Use `C:\o3de-build\android_apk` for `%O3DE_BUILD_ROOT%`</li><li>When running `generate_android_project.py` command use `LOOSE` for `%O3DE_ANDROID_ASSET_MODE%`</li><li>When running `deploy_android.py` command use `APK` for `%O3DE_ANDROID_DEPLOY_TYPE%`</li></ol></ol> | <ul><li>Binaries will be inside the android APK package. Unzip `O3DE_BUILD_ROOT/app/build/outputs/apk/profile/app-profile.apk` and verify `lib\arm64-v8a` folder contains `.so`.</li><li>Assets will be inside the android APK package. Unzip `O3DE_BUILD_ROOT/app/build/outputs/apk/profile/app-profile.apk` and verify `assets` folder contains all the assets from `DefaultProject\Cache\android`.</li><li>Execution's log and user folder will be generated in device's sdcard (`/sdcard/Android/data/com.o3de.DefaultProject/files/user/`).</li></ul> |
```

Comment:
I would remove specification of paths such as `C:\o3de-build\android` as that assumes a certain hard drive structure.

Furthermore it is Windows specific(even the use of environment variables using "%O3DE_BUILD_ROOT%")

Reply:
I recommend near the top of the workflow section of this document specifying some path aliases that will be used throughout.

Such as 
```md

<project-root> = path of Default Project root folder
<engine-root> = path of O3DE engine root folder
<o3de-build-root> = path specified as the build folder to generate_android_project (ex. <engine-root>/build/android_apk). Any folder can be used for this path.
```

Also the `lib\arm64-v8a` should be updated to use a forward slash so it becomes `lib/arm64-v8a`

Reply:
>I would remove specification of paths such as C:\o3de-build\android as that assumes a certain hard drive structure.

To make this document simpler to step says "Follow the steps from https://www.o3de.org/docs/user-guide/platforms/android/generating_android_project_windows/", and indicates alterations from that page, that page does use `C:\o3de-build` and that's is why it's indicating to change it to be clear.

>I recommend near the top of the workflow section of this document specifying some path aliases that will be used throughout.

It's the same issue, this page does not contain all the steps, it refers to the other page, it's more clear if it's directly using the same terms as that page.